docs/getting-started/your-first-chart.md

```diff
@@ -67,6 +67,8 @@ We will start off by creating a simple bar chart.
         <script type="text/javascript" src="/js/themes/gray.js"></script>
     ```
 
+    It is also possible to style the chart with CSS using [styled mode](https://www.highcharts.com/docs/chart-design-and-style/style-by-css).
```

Comment:
Just wondering, should this be point number 5 🤔?

Reply:
I was thinking of as an extension of step 4 (therefore the indentation).  Maybe it should be rewritten as a sublist 🤔

Reply:
Alright, both options make sense to me @goransle , feel free to rerequest review either way!